Tabloid/Repositories/PostRepository.cs

```diff
@@ -131,6 +130,52 @@ OUTPUT INSERTED.ID
             }
         }
 
+        public void AddPostReaction(PostReaction postReaction)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO PostReaction (PostId, ReactionId, UserProfileId)
+                                        OUTPUT INSERTED.ID
+                                        VALUES (@postId, @reactionId, @userProfileId)";
+                    cmd.Parameters.AddWithValue("@postId", postReaction.PostId);
+                    cmd.Parameters.AddWithValue("@reactionId", postReaction.ReactionId);
+                    cmd.Parameters.AddWithValue("@userProfileId", postReaction.UserProfileId);
```

Comment:
switch these with DbUtils.AddParameter and you're golden

Reply:
donezo